doc/specs/vulkan/validity/protos/vkCmdCopyBuffer.txt

```diff
@@ -16,8 +16,8 @@ endif::doctype-manpage[]
 * This command must: only be called outside of a render pass instance
 * The value of pname:regionCount must: be greater than `0`
 * Each of pname:commandBuffer, pname:srcBuffer and pname:dstBuffer must: have been created, allocated or retrieved from the same sname:VkDevice
-* The sum of the pname:srcOffset and pname:size members of a given element of pname:pRegions must: be less than or equal to the size of pname:srcBuffer
-* The sum of the pname:dstOffset and pname:size members of a given element of pname:pRegions must: be less than or equal to the size of pname:dstBuffer
+* The sum of the pname:srcOffset and pname:copySize members of a given element of pname:pRegions must: be less than or equal to the size of pname:srcBuffer
```

Comment:
Note: this is not an intended modification, but leftovers from a previous commit that did not ```make full_install```